client/public/index.html

```diff
@@ -2,42 +2,19 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/choicely.png" />
```

Comment:
I think svg or ico would be a better format

Reply:
The png is only for my favicon therefore i used this format there. But in general you are right and i try to use svg were ever i can 😇